src/debug.c

```diff
@@ -1600,8 +1598,10 @@ void doFastMemoryTest(void) {
                 "Fast memory test PASSED, however your memory can still be broken. Please run a memory test for several hours if possible.\n");
         }
     }
-#endif
 }
+#else
+void doFastMemoryTest(void) {}
+#endif
```

Comment:
```suggestion
#endif /* HAVE_PROC_MAPS */
```
How about adding a comment

Reply:
thanks, done